duckduckgo/Android
# Add light widget

Task/Issue URL: https://app.asana.com/0/361428290920652/825447419911129
Tech Design URL: N/A

**Description**:
Adds a light widget to the app

**Steps to test this PR**:

**TEST 1: Upgrade path**
1. Ensure that the widget is already installed
1. Install the new version of the app
1. Ensure the original widget works

**TEST 2: General**
1. Add the light widget to your home screen
1. Tap it and ensure it launches the app

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
